go.mod

```diff
@@ -1,53 +1,68 @@
-module github.com/golang-migrate/migrate/v4
```

Comment:
Don't change the module name.
Please open dependency updates in a separate PR.
Also, did you run `go mod tidy`?